igor-web/config/igor.yml

```diff
@@ -7,3 +7,11 @@ spinnaker:
     pollInterval: 30
     
 endpoints.health.sensitive: false
+
+#travis:
+#  enabled: true
+#  masters:
+#    - name: travis-ci
+#      baseUrl: https://travis-ci.org
+#      address: https://api.travis-ci.org
+#      githubToken: XXXXXX
```

Comment:
would like to see this in the config in https://github.com/spinnaker/spinnaker as well